app/tracing.ts

```diff
@@ -0,0 +1,227 @@
+import { BatchSpanProcessor, SamplingDecision, SamplingResult } from '@opentelemetry/sdk-trace-base';
+import { Resource } from '@opentelemetry/resources';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { AWSXRayIdGenerator } from '@opentelemetry/id-generator-aws-xray';
+import { Attributes, Context, Link, SpanKind, trace } from '@opentelemetry/api';
+import { AWSXRayPropagator } from '@opentelemetry/propagator-aws-xray';
+import env from './util/env';
+import { random } from 'lodash';
+import { AwsInstrumentation } from '@opentelemetry/instrumentation-aws-sdk';
+
+
+
+// Register instrumentation libraries for Node.js auto-instrumentation and capturing of
+// outgoing HTTP calls
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations(),
+    new HttpInstrumentation(),
+    new AwsInstrumentation({}),
+  ],
+});
+
+const resource =
+  Resource.default().merge(
+    new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: env.harmonyClientId,
+      [SemanticResourceAttributes.SERVICE_VERSION]: '0.1.0',
```

Comment:
I see on our home page `[v 0.0.0](https://harmony.earthdata.nasa.gov/versions)`. I'm wondering where the 0.0.0 comes from and if we should reference the same version info rather than hardcode.

Reply:
Done.

Reply:
import version from '../util/version';